simulation/common/helpers.py

```diff
@@ -922,6 +905,27 @@ def PJWHash(key: Union[np.ndarray, list, set, str, tuple]) -> int:
     return Hash & 0x7FFFFFFF
 
 
+def parse_coordinates_from_kml(coords_str: str) -> np.ndarray:
+    """
+
+    Parse a coordinates string from a XML (KML) file into a list of coordinates (2D vectors).
+    Requires coordinates in the format "39., 41., 0  39., 40., 0" which will return [ [39., 41.], [39., 40.] ].
```

Comment:
By the way you describe the format here, your code doesn't look like it would work.

Looking at the .kml file I see the correct formatting is "39.,41.,0 39.,40.,0"